CODEOWNERS

```diff
@@ -237,12 +237,9 @@
 /drivers/wifi/                            @jukkar @tbursztyka @pfalcon
 /drivers/wifi/eswifi/                     @loicpoulain
 /dts/arc/                                 @vonhust @ruuddw @iriszzw
-/dts/arm/atmel/sam4e*                     @nandojve
 /dts/arm/atmel/samr21.dtsi                @benpicco
 /dts/arm/atmel/sam*5*.dtsi                @benpicco
-/dts/arm/atmel/same70*                    @nandojve
-/dts/arm/atmel/samv71*                    @nandojve
-/dts/arm/atmel/                           @galak
+/dts/arm/atmel/                           @galak @nandojve
```

Comment:
This seems wrong - the last pattern match has preference so this will supersede @benpicco 

Honestly, I don't think there's so much need for clean up here - CODEOWNERS file is meant to provide fine granularity in assigning reviews.